src/lib/CollisionPrevention/CollisionPrevention.hpp

```diff
@@ -68,13 +68,12 @@ class CollisionPrevention : public ModuleParams
 
 	bool is_active() { return _param_mpc_col_prev_d.get() > 0; }
 
-	void modifySetpoint(matrix::Vector2f &original_setpoint, const float max_speed);
+	void modifySetpoint(matrix::Vector2f &original_setpoint, const float max_speed, const float max_acc);
 
 private:
 
 	bool _interfering = false;		/**< states if the collision prevention interferes with the user input */
```

Comment:
```suggestion
	bool _interfering{false};		/**< states if the collision prevention interferes with the user input */
```

Please use C++11 brace-initialization (for all the members, not only this one :D )